### PR TITLE
Another Round of Editing

### PR DESCRIPTION
**Pull request by agracey**

### CODE REVIEW

**Comment by agracey** [comment]
@timirnich I think this  is as good as I can do for now.  I'll turn TODOs into issues in the next PR.

Also,  what should next steps be? I'll see if Troy has any ideas

File: content/cli.md [hunk 1]
```diff
@@ -119,33 +120,27 @@ Space:          samples
   
 ## Organizations and Spaces
 
-The applications and users of a Cloud Foundry platform are split into Organizations and Spaces. This gives a way to built a multi-tenant environment with minimal headache since each of these organizations and spaces can be given it's own resource quotas and access controls.
+All applications (and users) on a Cloud Foundry platform are split into Organizations and Spaces. This gives a way to build a multi-tenant environment with minimal headache since each organization and space can be given it's own resource quota and access control.
 
-As part of your free trial, you have administrator rights for your own organization with a few different spaces by default. By default, there are a few applications running in the "samples" space (selected at login).
+As part of your free trial, you have administrator access for a personal organization with a few spaces. By default, there are a few applications running in the "samples" space (selected at login).
```
Comment (timirnich) [inline content/cli.md]:
```suggestion
As part of your developer sandbox, you have administrator access for a personal organization with a few spaces. By default, there are a few applications running in the "samples" space (selected at login).
```

File: content/quickstart.md [hunk 2]
```diff
@@ -62,7 +62,8 @@ The Stratos Web UI can be accessed at https://stratos.cap.explore.suse.dev. Cred
 
 ## Where to get help
 
-If you get stuck or run into trouble, you can reach us in the #developersandbox channel on our Discord server, or write an email to devcommunity@suse.com. 
+If you get stuck or run into trouble, you can check out the [FAQ](/faq/), reach us in the #developersandbox channel on our Discord server, or write an email to devcommunity@suse.com.
```
Comment (timirnich) [inline content/quickstart.md]:
```suggestion
If you get stuck or run into trouble, you can check out the [FAQ](/faq/), reach us in the #developersandbox channel on our Discord server, or write an email to developer@suse.com.
```

**Review by timirnich — CHANGES_REQUESTED** [review]
All good, two small fixes and then I'd say this is perfectly fine and yet another good step forward. Thanks!

**Pull request merged**